### DISCUSSION TITLE
Plugin to allow variants of a test to be run using tags needed

### DISCUSSION THREAD

**Discussion by PanosBant** [reqnroll/Reqnroll#221] [Ideas]
Hello ,
specflow had a great plugin to allow variants of a test to be run using tags. 
I wonder if you have something similar to SpecFlow.Contrib.Variants plugin or if you plan something for that since it is a blocking point to continue migration from specflow.

**Comment by gasparnagy** [comment]
This sounds useful. We don't have direct short-term plans to have built-in support for this. Not because they are not useful, but because we don't have too much dev capacity. 

The easiest would be if someone would port that plugin to Reqnroll. It seems to have a nice codebase, so it should not be very hard. I don't know if @TotalTest is interested in doing this, but if you @PanosBant can help, I think you can coordinate this.

**Reply by TotalTest** [reply]
I do plan to port the plug in to Reqnroll but I’ll need to find some time to dedicate to it, so cannot yet put an accurate timeline on this.

**Comment by ClumsyPenguin** [comment]
@gasparnagy @TotalTest 
Since yesterday SpecFlow seems to be [no longer open source](https://www.reddit.com/r/dotnet/comments/1hgyxrp/specflow_is_no_longer_open_source/?share_id=3jHQAdzyyVQb1YvMWhdeA&utm_medium=ios_app&utm_name=ioscss&utm_source=share&utm_term=1)? 

The repositories seems to be made private https://github.com/SpecFlowOSS

We currently relying heavily on this plugin to support multi-tenancy in our testing framework. I tried porting the plugin, but ran into some issues with the CodeDomHelper class. Is anyone still looking out for this plugin?

**Reply by gasparnagy** [reply]
@ClumsyPenguin I will definitely have no time for making the plugin port before February, but if you started and have found concrete issues, I am happy to help. Please post the problems you have faced here or even better to a new issue ticket and we will help.
